test/integration/sparql-test.js

```diff
@@ -21,7 +21,8 @@ describe('a query path with a path expression handler', () => {
     pathExpression: new PathExpressionHandler(),
     add: new InsertFunctionHandler(),
     delete: new DeleteFunctionHandler(),
-    properties: new PropertiesHandler(),
+    properties: new PredicatesHandler(),
+    predicates: new PredicatesHandler(),
```

Comment:
is it really necessary to keep both? wouldn't it be enough to keep only "predicate"?

Reply:
The audience that LDflex targets might not be as familiar with the term "predicate".